src/Stepper/Step.jsx

```diff
@@ -0,0 +1,336 @@
+import React, {PropTypes} from 'react';
+
+import {
+  TouchRipple,
+} from '../ripples';
+
+import Avatar from '../avatar';
+import Mixins from '../mixins';
+
+import {getMuiTheme} from '../styles';
+
+const Step = React.createClass({
+  propTypes: {
+    children: PropTypes.node,
+
+    /**
+     *Override the inline-styles of div which contains all the children include control button groups
+     */
+    childrenWrapperStyle: PropTypes.object,
+    /**
+     *Override the inline-styles of connector line
+     */
+    connectorLineStyle: PropTypes.object,
+    /**
+     *An array of node for handling moving or canceling steps
+     */
+    controlButtonsGroup: PropTypes.arrayOf(PropTypes.node),
```

Comment:
@namKolo Right now in the docs I noticed the two buttons don't have any spacing in between them when I think they probably should.

I wonder what the best API is, I suppose a few options are:
1. Leave it as is, and the developer should just add padding to the control buttons themselves based on the spacing they want
2. Change this prop to accept one node (instead an array of nodes) that represents the area for the controlButtons and they style it themselves (I suppose if we do this, we could remove the `controlButtonsGroupWrapperStyle` because they could apply that themselves too)

Any other ideas?